iroha_data_model/src/query.rs

```diff
@@ -78,14 +79,8 @@ pub struct QueryResult(pub Value);
 impl QueryRequest {
     /// Constructs a new request with the `query`.
     pub fn new(query: QueryBox) -> Self {
-        QueryRequest {
-            timestamp: SystemTime::now()
-                .duration_since(SystemTime::UNIX_EPOCH)
-                .expect("Failed to get System Time.")
-                .as_millis()
-                .to_string(),
-            query,
-        }
```

Comment:
I think the timestamp that we had here is okay, as it is a unix timestamp and therefore it is universal. We just don't want to convert it to string. Use u64 as in `creation_time` in transaction.

Reply:
Maybe we can use scale's `CompactEncoding` for this, but I am not sure on this.